src/chapters/configuring.xml

```diff
@@ -3201,13 +3224,9 @@ color9: rgb:80/00/80</programlisting>
                 </informaltable>
             </section>
             <section xml:id="ssh-command">
```

Comment:
If this section is no longer specific to running a command, the ID should be updated accordingly, as it's this ID that will be used to link to specific sections.

Reply:
Yeah, I have no clue what I was doing, here.  My only guess is that I didn't have enough :coffee: before writing this...